docs: Update rule-proposal to add link to tag assignment (#3184)
dequelabs/axe-core: doc/rule-proposal.md

#### Tags

> Indicate which tags the rule should use. See the documentation on [how we assign tags](https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#axe-core-tags).

Reviewer: ```suggestion
Indicate which tags the rule should use. More information available in the [how we assign tags documentation](https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#axe-core-tags).
```
